### ISSUE TITLE
[Bugs] Failed to build the main branch of amf

### ISSUE THREAD

**Issue by s5uishida** [free5gc/free5gc#188]
Hi @free5gc-org 

After git clone free5gc, I ran the following script to change each NFs branch to `main` and tried to build all NFs,
```
#!/usr/bin/env bash

NF_LIST="nrf amf smf udr pcf udm nssf ausf upf n3iwf"

for NF in ${NF_LIST}; do
    cd NFs/${NF}
    git checkout main
    cd ../..
done

cd webconsole
git checkout main
```
but failed to build `amf` as follows.
```
# make amf
Start building amf....
cd NFs/amf && \
CGO_ENABLED=0 go build -ldflags "-X github.com/free5gc/version.VERSION=v3.0.5 -X github.com/free5gc/version.BUILD_TIME=2021-03-16T12:56:40Z -X github.com/free5gc/version.COMMIT_HASH=+941d6ac -X github.com/free5gc/version.COMMIT_TIME=2021-03-12T00:52:26Z" -o /root/free5gc/bin/amf amf.go
# github.com/free5gc/amf/ngap/message
ngap/message/send.go:16:4: undefined: ngaplog
make: *** [Makefile:44: bin/amf] Error 2
```
I was able to build all other NFs except `amf`.

I would be very glad if you could fix it.

Best regards,

--Shigeru

**Comment by free5gc-org** [comment]
@s5uishida 
It is already fixed by commit: eba10b4ff183df9c724237f479781dbe0953f720.
You can get the latest to build again.

**Comment by s5uishida** [comment]
Hi @free5gc-org 

Thanks you very much. I was able to build `main` branch of `amf`.
